packages/components/src/FileField/FileField.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+import RAC, { DropZone } from 'react-aria-components';
+import { useLocale } from '@react-aria/i18n';
+import {
+  FieldBase,
+  type FieldBaseProps,
+  FileTrigger,
+} from '@marigold/components';
+import { WidthProp, useClassNames } from '@marigold/system';
+import { FileFieldItem } from './FileFieldItem';
+import { isFileDropItem, normalizeAndLimitFiles } from './fileUtils';
+
+type RemovedProps =
+  | 'className'
+  | 'style'
+  | 'children'
+  | 'isDisabled'
+  | 'isRequired';
+
+export interface FileFieldProps
+  extends Omit<RAC.DropZoneProps, RemovedProps>,
+    Pick<FieldBaseProps<'label'>, 'label'> {
```

Comment:
isn't it "input" what will be targeted by a ref passed to the `FileTrigger` stuff?